src/components/layout/nav/nav-sticky.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { DesktopWrapper, MobileWrapper } from './styles/nav-styles'
+import NavDesktop from './nav-desktop'
+import NavMobile from './nav-mobile'
+import { SectionContainer } from 'components/containers'
+import device from 'themes/device'
+import { isLoggedIn } from 'common/utility'
+import { CFDWarning } from 'components/layout'
+
+const Section = styled(SectionContainer)<{ background?: boolean }>`
+    background-color: ${({ background }) => (background ? 'transparent' : 'var(--color-black)')};
+    width: 100%;
+    position: fixed;
+    z-index: 5;
+    top: 0;
+    padding: 4px 1%;
+    height: 7.2rem;
+    @media ${device.tabletL} {
+        padding: 4px;
+        height: 64px;
+    }
+    @media ${device.mobileL} {
+        padding: 4px 0;
+    }
+`
+
+type NavStickyProps = {
+    is_ppc: boolean
+    hide_signup_login?: boolean
+    no_language?: boolean
+}
+
+const NavSticky = ({ is_ppc, hide_signup_login, no_language }: NavStickyProps) => {
+    const [is_logged_in, setLoggedIn] = useState(true)
+    const [prevScrollPos, setPrevScrollPos] = useState(0)
```

Comment:
our naming convention for states are like the one in the upper line
prev_scroll_pos 
I know this is odd :-D